src/pages/index.js

```diff
@@ -5,17 +5,107 @@ import Section from "../components/Section.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import UserInfo from "../components/Userinfo.js";
-import { initialCards } from "../utils/constants.js";
+import Api from "../components/Api.js";
+import PopupWithConfirm from "../components/PopupWithConfirm";
 import { config } from "../utils/constants.js";
+import {
+  profileEditButton,
+  inputName,
+  inputDescription,
+  cardAddButton,
+  avatarEditButton,
+} from "../utils/constants.js";
+
+/*
+  ┌─────────────────────────────────────────────────────────────────────────┐
+  │ API INSTANCE                                                            │
+  └─────────────────────────────────────────────────────────────────────────┘
+ */
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "5e1b3b9f-b018-4d2b-bc9a-2c16cad524b1",
+    "Content-Type": "application/json",
+  },
+});
+
+let cardSection;
+let userId;
+
+/*
+  ┌─────────────────────────────────────────────────────────────────────────┐
+  │ POPUP ELEMENTS                                                          │
+  └─────────────────────────────────────────────────────────────────────────┘
+ */
+
+const imagePreviewPopup = new PopupWithImage("#modal-preview-image");
+
+const addCardPopup = new PopupWithForm("#modal-add-card", handleCardFormSubmit);
+
+const editProfilePopup = new PopupWithForm(
+  "#modal-profile-edit",
+  handleProfileFormSubmit
+);
 
-// MODAL ELEMENTS
+const cardDeletePopup = new PopupWithConfirm("#modal-card-delete");
 
-const profileEditButton = document.querySelector("#profile-edit-button");
-const inputName = document.querySelector("#form-input-name");
-const inputDescription = document.querySelector("#form-input-description");
-const cardAddButton = document.querySelector("#card-add-button");
+const avatarEditPopup = new PopupWithForm(
+  "#modal-avatar-edit",
+  handleAvatarFormSubmit
+);
 
-// VALIDATION
+const userInfo = new UserInfo(
+  document.querySelector(".profile__name"),
+  document.querySelector(".profile__subtitle"),
+  document.querySelector(".profile__photo")
+);
+
+/*
+  ┌─────────────────────────────────────────────────────────────────────────┐
+  │ LOAD USER INFO FROM SERVER                                              │
+  └─────────────────────────────────────────────────────────────────────────┘
+ */
+
+api
+  .getUserInfo()
```

Comment:
You need to combine the requests of the user's info and the cards together in `Promise.all` because it might happen so that the `_id` has not been received yet but the cards have and your cards will be rendered not correctly. 
More info is here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

```js

Promise.all([getUserInfo(), getCards()])
// destructure the response
  .then(([userData, cards]) => {
      // set all the data
  })
  .catch(err => {
    // catch possible errors
  });

```